### ISSUE TITLE
salt.utils.dns.lookup fails with host, nslookup method when servers specified

### ISSUE THREAD

**Issue by yosnoop** [saltstack/salt#51249]
### Description of Issue/Question
When salt.utils.dns.lookup executed with _host_ or _nslookup_ method and _servers_ option specified, the following error occurs. 
`Passed invalid arguments: _multi_srvr() got multiple values for keyword argument 'server'.`

### Setup
(Please provide relevant configs and/or SLS files (Be sure to remove sensitive info).)
Clean install on Arch Linux. No additional configuration.

### Steps to Reproduce Issue
(Include debug logs if possible and relevant.)
```
$ cat <<END >lookup.py
> import salt.utils
> def run():
>     result = salt.utils.dns.lookup('google.com', 'soa', method='nslookup', servers=['8.8.8.8','8.8.4.4'])
> END
$ salt-call lookup.run 

Passed invalid arguments: _multi_srvr() got multiple values for keyword argument 'server'.

Usage:
None
$ 
```

### Versions Report
(Provided by running `salt --versions-report`. Please also mention any differences in master/minion versions.)
```
$ salt --versions-report 
Salt Version:
           Salt: 2018.3.3
 
Dependency Versions:
           cffi: 1.11.5
       cherrypy: Not Installed
       dateutil: Not Installed
      docker-py: Not Installed
          gitdb: Not Installed
      gitpython: Not Installed
          ioflo: Not Installed
         Jinja2: 2.10
        libgit2: Not Installed
        libnacl: Not Installed
       M2Crypto: 0.30.1
           Mako: Not Installed
   msgpack-pure: Not Installed
 msgpack-python: 0.6.0
   mysql-python: Not Installed
      pycparser: 2.19
       pycrypto: 3.7.2
   pycryptodome: Not Installed
         pygit2: Not Installed
         Python: 2.7.15 (default, Jan 10 2019, 23:20:52)
   python-gnupg: Not Installed
         PyYAML: 3.13
          PyZMQ: 17.1.2
           RAET: Not Installed
          smmap: Not Installed
        timelib: Not Installed
        Tornado: 5.1.1
            ZMQ: 4.3.1
 
System Versions:
           dist:   
         locale: UTF-8
        machine: x86_64
        release: 4.20.0-arch1-1-ARCH
         system: Linux
        version: Not Installed
```

**Comment by dwoz** [comment]
@yosnoop Thanks for reporting this and working on a fix!

**Comment by yosnoop** [comment]
Closing as it merged to 2018.3.
